lib/dotcom/trip_plan/open_street_map_reconciler.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Dotcom.TripPlan.OpenStreetMapReconciler do
+  @moduledoc """
+  There are some locations that are fairly close to transit stops, but
+  for which all of the OpenStreetMap paths are not public. The
+  Massachusetts State House is a good example of this. The building
+  isn't publicly accessible according to OpenStreetMaps data, and
+  that's correct, because all of the entries into it are gated with
+  security guards. That said, plenty of people need to plan trips to
+  the State House.
+
+  This module is responsible for reconciling locations that aren't
+  publicly accessible, but that should still be possible to plan trips
+  to and from, to a sensible lat/long that can be used in trip plans,
+  e.g. the public side of a security entrance.
+  """
+
+  alias Dotcom.TripPlan.InputForm
+
+  @state_house_location %{latitude: 42.35861, longitude: -71.06297}
+
+  @state_house_geofence %Geo.Polygon{
+    coordinates: [
+      [
+        {-71.06428959524202, 42.35929686684236},
```

Comment:
```
{-71.06429, 42.35930}
```
Please trim to fewer significant figures 😅 Matching your state house point seems good!

Reply:
Kinda wish you'd built a simpler polygon so this wasn't sooo long!